akka-stream/src/main/scala/akka/stream/javadsl/MergeLatest.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.javadsl
+
+import akka.stream.UniformFanInShape
+import akka.stream.scaladsl.MergeLatest
+import akka.stream.stage.GraphStage
+
+import scala.collection.JavaConverters._
+import scala.reflect.ClassTag
+
+/**
+ * MergeLatest joins elements from N input streams into stream of lists of size N.
+ * i-th element in list is the latest emitted element from i-th input stream.
+ * MergeLatest emits list for each element emitted from some input stream,
+ * but only after each stream emitted at least one element
+ *
+ * '''Emits when''' element is available from some input and each input emits at least one element from stream start
+ *
+ * '''Backpressures when''' downstream backpressures
```

Comment:
Is that really true? I mean, the purpose is to have the _latest_ merged, which means that we do not want to backpressure but drop the elements while one has not pulled; otherwise such merge would not be any different from a normal Merge